sfdoc/publish/tasks.py

```diff
@@ -138,16 +141,19 @@ def create_drafts(bundle, html_files, path, salesforce_docset, s3):
             html_file.replace(path + os.sep, ''),
         )
         _scrub_and_analyze_html(
-            html_file, path, article_image_map, images, url_map, problems
+            bundle, html_file, path, article_image_map, images, url_map, problems
         )
 
+
     # check for duplicate URL names
     problems.extend(_find_duplicate_urls(url_map))
 
 
     # give up if there are problems before we start making database
     # objects
     if problems:
+        for problem in problems:
+            logger.info("ERROR! %s", problem)
         raise SfdocError(repr(problems))
```

Comment:
Does this not end up showing the problems twice?

Reply:
Logs and errors actually end up in different places and errors are hard to read because they are all on one super-long line.

Our error output is a stringification of an error object containing a stringification of a list of strings. In short: hard to read.